todo_project/settings/production.py

```
@@ -4,3 +4,9 @@
 DEBUG = False
 
 ALLOWED_HOSTS = os.getenv("ALLOWED_HOSTS").split(",")
+
+SPECTACULAR_SETTINGS.update({
+    "SWAGGER_UI_SETTINGS": {
+        "url": "/todo/api/schema",
+    },
+})
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)